sdk/storage/test-resources-post.ps1

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This script is used to set up CORS rules for queues after storage account creation by New-TestResources.ps1
```

Comment:
Also worth mentioning that `the ARM template doesn't allow setting CORS rules for queue service, whereas it does for blob and file.` and `this script is meant to be a workaround` 
??